docs/tutorials/programmability/ansible/intent-based-management/config.md

```diff
@@ -0,0 +1,614 @@
+    |              |          | 192.168.0.3     | disabled  | 2/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65002   | established |
+    |              |          | 192.168.0.5     | disabled  | 2/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65003   | established |
+    |              |          | 192.168.0.7     | disabled  | 2/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65004   | established |
+    |              |          | 192.168.255.1   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |              |          | 192.168.255.2   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |              |          | 192.168.255.3   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |              |          | 192.168.255.4   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |--------------+----------+-----------------+-----------+-----------+----------------+---------+---------------+----------+---------+-------------|
+    | clab-4l2s-s2 | default  | 192.168.1.1     | disabled  | 5/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65001   | established |
+    |              |          | 192.168.1.3     | disabled  | 5/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65002   | established |
+    |              |          | 192.168.1.5     | disabled  | 5/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65003   | established |
+    |              |          | 192.168.1.7     | disabled  | 5/1/8     | pass-all       | leafs   | pass-all      | 65100    | 65004   | established |
+    |              |          | 192.168.255.1   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |              |          | 192.168.255.2   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |              |          | 192.168.255.3   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    |              |          | 192.168.255.4   | 0/0/0     | disabled  | pass-evpn      | overlay | pass-evpn     | 100      | 100     | established |
+    +-------------------------------------------------------------------------------------------------------------------------------------------------+
+    ```
+
+From the BGP Peers output, you can see that we use eBGP for underlay routing (ASNs 65001-65004 for the leafs and ASN 65100 for the spines) and that the iBGP mesh for the overlay (ASN 100) is established through route-reflector (RR) functionality on the spines.
```

Comment:
ideally we should have a diagram here explaining what is configured at this point